api/app/models.py

```diff
@@ -0,0 +1,39 @@
+from app import db
+from sqlalchemy.dialects.postgresql import JSON, ARRAY
+
+class election(db.Model):
+    __tablename__ = 'election'
+    id = db.Column(db.Integer, primary_key = True)
+    name = db.Column(db.String)
+    description = db.Column(db.String)
+
+    questions = db.Column(ARRAY(db.Integer))
+
+    hasVoted = db.Column(JSON) # ids of users who have voted in this election w/ timestamp
+
+    active = db.Column(db.Boolean)
+    creator = db.Column(db.Integer)
+    deadline = db.Column(db.DateTime)
+
+class question(db.Model):
+    __tablename__ = 'question'
+
+    id = db.Column(db.Integer, primary_key = True)
+    question = db.Column(db.String)
+    votes = db.Column(JSON) # Options and results
+    voteType = db.Column(db.String) # Proportional vs SPTP vs Ranked
```

Comment:
We will want to limit the options for voting types later on here, but for now we'll leave this as-is (we're not 100% certain what the options will be internally)